### PR TITLE
Add OP Sepolia

### PR DESCRIPTION
**Pull request by zainbacchus**
Add OP Sepolia 

See: https://docs.optimism.io/op-networks

### CODE REVIEW

**Review by macvts — COMMENTED** [review]
just some reminder but I saw op sepolia testnet available on both blockscout and etherscan, if blockscout is the main explorer you wish new user to go, then its fine.

File: _data/chains/eip155-11155420.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://optimism.io",
+  "shortName": "opsep",
+  "chainId": 11155420,
+  "networkId": 11155420,
+  "explorers": [
+    {
+      "name": "opscout",
+      "url": "https://optimism-sepolia.blockscout.com/",
```
Comment (macvts) [inline _data/chains/eip155-11155420.json:18]: after checking the failure 
need to cut the / off 
for the explorer url 

Screenshot 2566-11-22 at 22 31 59

**Comment by Daddyk1230** [comment]
> pls am new here can someone teach me how to use this app

**Comment by macvts** [comment]
@Daddyk1230 by mentioning app, I prefer you to try browser version first so there will be more of the details showing and get used to this by https://docs.github.com/en/get-started

**Comment by zainbacchus** [comment]
> @macvts should I open a new PR to get this in? It has been 2 weeks and no fix from original poster

Thank you!

**Pull request closed without merging**